internal/template/templates/common/layout.html

```diff
@@ -36,7 +36,7 @@
 
     {{ if and .user .user.Stylesheet }}
     {{ $stylesheetNonce := nonce }}
-    <meta http-equiv="Content-Security-Policy" content="default-src 'self'; img-src * data:; media-src *; frame-src *; style-src 'self' 'nonce-{{ $stylesheetNonce }}'">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' {{ .contentSecurityPolicy }}; img-src * data:; media-src *; frame-src *; style-src 'self' {{ .contentSecurityPolicy }} 'nonce-{{ $stylesheetNonce }}'">
```

Comment:
if you mean why not:
```html
<meta http-equiv="Content-Security-Policy" content="{{ .contentSecurityPolicy }}">
```
then it's because of nonce - we need to keep it. so my change only whitelists domains in CSP.

it's also possible to do:

```html
<meta http-equiv="Content-Security-Policy" content="{{ .contentSecurityPolicy }} 'nonce-{{ $stylesheetNonce }}'">
```
but I think it's error-prone, as user would need to always keep `style-src` at the end

Reply:
updated to replace the whole value. but updated version still has limitation: user's value should contain `%s`, where nonce will be placed.
this may be solved for example by checking whether user's value contain `%s`. if no - output meta tag without nonce. do you think it's a good approach?